app/models/alchemy/page/page_scopes.rb

```diff
@@ -27,7 +27,7 @@ module Page::PageScopes
 
       # All not locked pages
       #
-      scope :not_locked, -> { where(locked: false) }
+      scope :not_locked, -> { where(locked_at: nil, locked_by: nil) }
```

Comment:
What happens if ONE of these two is `nil`? Is the page in lock-limbo? Do both get reliably reset when unlocking the page? I haven't seen code that does this.